cachito/web/models.py

```diff
@@ -1073,6 +1079,12 @@ def get_pkg_managers(cls, pkg_managers):
 
         return found_pkg_managers
 
+    @classmethod
+    @functools.cache
```

Comment:
This is new in python 3.9, my bad. I think `lru_cache(maxsize=None)` does the same thing.

Reply:
Add py38 to tox testenv as well?